AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminMediaGalleryAddSelectedSavedPreviewActionGroup.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminMediaGalleryAddSelectedSavedPreviewActionGroup">
```

Comment:
Please use `AdminSaveAdobeStockImagePreviewActionGroup` and `AdminMediaGalleryClickAddSelectedActionGroup` instead of this action group